doc/source/examples/incompressible-flow/2d-backward-facing-step/2d-backward-facing-step.rst

```diff
@@ -0,0 +1,348 @@
+
+However, for :math:`Re \geq 700`, convergence can be quite difficult to obtain while doing a steady state simulation. In fact, as the Reynolds number increases, the problem becomes progressively stiffer to a point where the ``steady`` solver ultimately fails. With that in mind, the case can be solved as a transient problem until the steady state solution is obtained. This can be achieved with the ``method = steady_bdf`` parameter.
+
+.. code-block:: text
+
+	subsection simulation control
+	  set method = steady_bdf
+	  set stop tolerance = 1e-6
+	  set time step                     = 0.005
+	  set adapt                         = true
+	  set max cfl                       = 1e6
+	  set adaptative time step scaling = 1.2
+	  set number mesh adapt = 0
+	  set output name = backward_facing_step_output
+	  set output frequency = 1
+	  set subdivision = 1
+	  set output boundaries = false
+	end
+  
+``stop tolerance``, ``time step``, ``adapt``, ``max cfl`` and ``adaptive time step scaling`` are parameters that control the pseudo-steady simulation. Since this problem can be quite demanding, a relatively high tolerance has been chosen as an input for ``stop tolerance``. This choice is however not without its consequences : the :math:`x_r` parameter might be slightly underestimated. 
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+In this problem, the Reynolds number is defined as follows : 
+
+.. math::
+	Re_{D_h} = \frac{u D_h}{\nu} = \frac{2uh}{\nu}
+	
+where :math:`h` is the step height, :math:`D_h = 2h` is the hydraulic diameter and :math:`\nu` the kinematic viscosity.
+
+In addition, unit values of :math:`u` and :math:`h` are chosen in the goal of obtaining an adimensional problem.
+
+.. math::
+	Re_{D_h} = f(\nu) = \frac{2}{\nu}
+	
+Consequently, the physical properties are defined as follows : 
+
+.. code-block:: text
+	
+	subsection physical properties
+	  set number of fluids = 1
+	  subsection fluid 0
+		set kinematic viscosity = 0.01 # Re_h=2/nu
+	  end
+	end
+	
+.. note::
+	The ``kinematic viscosity`` is the only parameter that changes coherently with :math:`Re` (for example, to obtain :math:`Re = 100`, ``kinematic viscosity = 0.02`` can be used as input). 
+
+Mesh
+~~~~
+
+.. code-block:: text
+
+	subsection mesh
+	  set type = gmsh
+	  set file name = ../backward_facing_step.msh
+	end
+	
+The mesh features quad elements as well as unit step and inlet heights (:math:`h_{in}=h=1`). In that direction, the expansion ratio has been set to :math:`\beta=\frac{h_{out}}{h_{in}}=2` throughout the entirety of the simulations. Also, the inlet and outlet lengths should be long enough that they allow the formation of a fully developed flow and their respective end. Also, since a gmsh mesh file is used, the initial mesh should be as coarse, since these cells cannot be coarsened with the mesh adaptation algorithm.
```

Comment:
```suggestion
The mesh features quad elements as well as unit step and inlet heights (:math:`h_{in}=h=1`). In that direction, the expansion ratio has been set to :math:`\beta=\frac{h_{out}}{h_{in}}=2` throughout the entirety of the simulations. Also, the inlet and outlet lengths should be long enough that they allow the formation of a fully developed flow and their respective end. Also, since a ``gmsh`` mesh file is used, the initial mesh should be as coarse, since these cells cannot be coarsened with the mesh adaptation algorithm.
```